packages/devtools_app/lib/src/table.dart

```diff
@@ -740,6 +739,10 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
         }
       });
     });
+
```

Comment:
why isn't the selection notifier listener also tracked in the helper called from initState and didUpdateWidget?

Reply:
I didn't modify anything about the selection listener because it's not in the scope of this PR - will add a note to address